decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -382,6 +382,10 @@
   # Read more: https://docs.decidim.org/en/develop/configure/initializer#_content_security_policy
   config.content_security_policies_extra = {}
 
+  # This config parameter is used to set the default amount of time after which the unattached blobs
+  # are cleaned up. The default value is 60 minutes.
+  # config.clean_up_unattached_blobs_after = 60.minutes
+
```

Comment:
As I mentioned in another comment, I think we can drop this accessor

```suggestion
```

Reply:
done locally